src/diff/children.js

```diff
@@ -137,13 +114,23 @@ export function diffChildren(
 			}
 		}
 
-		oldVNode = oldVNode || EMPTY_OBJ;
+		const old = (oldVNode && oldVNode._node) || EMPTY_OBJ;
+		const oldVNodeDom = oldVNode && oldVNode._dom;
+		const oldRef = oldVNode && oldVNode.ref;
+		if (oldVNode == null || !childVNode._node) {
+			childVNode = createBackingNode(childVNode);
```

Comment:
I got stuck on this for a sec -  think we could use a second variable for the backing node so that it's clear which is which? Avoids accidentally trying to access backing node properties on a VNode.